apps/src/lib/kits/maker/boards/microBit/MBFirmataUpdater.js

```diff
@@ -0,0 +1,86 @@
+import {
+  MICROBIT_FIRMATA_V1_URL,
+  MICROBIT_FIRMATA_V2_URL,
+  MICROBIT_IDS_V1,
+  MICROBIT_IDS_V2,
+  MICROBIT_V1,
+  MICROBIT_V2
+} from '@cdo/apps/lib/kits/maker/boards/microBit/MicroBitConstants';
+import {DAPLink, WebUSB} from 'dapjs';
+import {getStore} from '@cdo/apps/redux';
+import {setMicrobitFirmataUpdatePercent} from '@cdo/apps/lib/kits/maker/microbitRedux';
+import {EventEmitter} from 'events'; // provided by webpack's node-libs-browser
+
+export default class MBFirmataUpdater extends EventEmitter {
```

Comment:
two nits:
1. I'd consider expanding `Microbit` in the name here, so if you were searching the file structure for  "microbit" the component's name would come up. (Though it doesn't matter because microbit is mentioned plenty in this file.)
2. I'd love to decide whether it's "Microbit" or "MicroBit" and keep it consistent. I see both "microBit" and "microbit" used in the PR. (But I'm also a stickler for this, and maybe no one else cares -- `weblab` vs. `webLab` drove me nuts while working on the test page.)

Reply:
For your first point, we already have `MBFirmataClient.js` and `MBFirmataWrapper.js`, so I was naming using that convention. But I'm definitely not beholden to the name. Lmk your thoughts considering the other 2 files. For context, `MBFirmataClient.js` is a copy of [this upstream version](https://github.com/microbit-foundation/microbit-firmata/blob/master/client/MBFirmataClient.js). A TODO to use the upstream and remove this locally.

And yes, we definitely need more consistency on that! I'll go ahead and change `MicrobitRedux.js` to `MicroBitRedux.js` and change constants I added in this PR to use ethis convention. Thanks!

Reply:
Keeping the file name is fine if that's a convention we use. These two were related because `MB` implies the "b" in `Micro Bit` is capital, so if we switch what we've added to use the `microBit` convention, that'll address both!